core/src/main/resources/hudson/security/Messages.properties

```diff
@@ -37,6 +37,7 @@ HudsonPrivateSecurityRealm.ManageUserLinks.Description=Create/delete/modify user
 HudsonPrivateSecurityRealm.CreateAccount.TextNotMatchWordInImage=Text didn''t match the word shown in the image
 HudsonPrivateSecurityRealm.CreateAccount.PasswordNotMatch=Password didn''t match
 HudsonPrivateSecurityRealm.CreateAccount.FIPS.PasswordLengthInvalid=Password must be at least 14 characters long
+HudsonPrivateSecurityRealm.CreateAccount.BCrypt.PasswordTooLong=Jenkins’ own user database currently only supports passwords of up to 72 bytes UTF-8 (72 basic ASCII characters, 24-36 CJK characters, or 18 emoji). Please use a shorter password.
```

Comment:
This is an example of a developer error message and not a user message.

For a user I would suggest something more like:

> Jenkins’ own user database only supports passwords up to 72 characters

> Maximum password length is 72 characters

> Password is too long

If you really want to tweak for UTF or emoji you could put a more specific error message in there.

Reply:
Thanks for the feedback. I acknowledge the problem in the Jira issue when I wrote

> longer than 72 bytes (UTF-8, so good luck explaining this requirement to users)

but I couldn't think of a great alternative, other than providing examples what this means (and why "72 characters" isn't it -- option 1 and 2 are simply wrong).

Something more vague like the following might work, as it captures the important parts while being overall less technical.

> usually 72 characters, fewer when using other characters, like Chinese characters or emoji

Reply:
https://github.com/jenkinsci/jenkins/pull/10626